npm-scripts.js

```diff
@@ -41,19 +47,66 @@ switch (task)
 
 	case 'postinstall':
 	{
-		if (!process.env.MEDIASOUP_WORKER_BIN)
-		{
-			buildWorker();
 
-			if (!process.env.MEDIASOUP_LOCAL_DEV)
+		if (!process.env.MEDIASOUP_LOCAL_DEV)
+		{
+			cleanWorker();
+		}
+		else if (!process.env.MEDIASOUP_WORKER_BIN)
+		{
+			// Attempt to download a prebuild binary
+			downloadPrebuild().catch((error) =>
 			{
-				cleanWorker();
-			}
+				console.error('Failed to download prebuilt binary, building instead', error.message);
```

Comment:
Those logs don't conform to the syntax of other console logs in same file. Also, why are you using then and catch instead of try/catch with await in the try block? We strongly promote that.

Reply:
Old habits I guess. This is the first instance of async/await in this file and top level await was added unflagged in node 14.8. Your package.json says you support >=16 so that change should be fine. I'll make the change.

Note that it will explicitly break earlier versions of node, which I just confirmed are out of maintenance, so probably fine.

Reply:
Yup :)

BTW I've pulled your branch and I'm some cosmetic changes so I'll ask you later to cherry-pick them in your branch. So let me please convert those to await syntax.

Reply:
Hmm. Today I learned something new about top level await (honestly never attempted it before). In order to convert the syntax to async/await I'd need to convert the whole file into ESM instead of CJS. 

So, renamed to `npm-scripts.mjs` and swap the `require` for `import` statements.

I'm happy to do so, but I wanted to check with you before I rewrite a lot.

Reply:
👍 Sorry, pausing for a few hours, I'll circle back around in the afternoon after meetings and your changes are done.